`src/lib/Makefile.in` has a broken libdir path.

It has 

```
cp $(NDPI_LIBS) $(DESTDIR)$(PREFIX)$(libdir)/

```

which is installing libraries into `/usr/usr/lib64/`, because in Autotools logic, `$libdir` must be absolute.

Just make it 
```
cp $(NDPI_LIBS) $(DESTDIR)$(libdir)/

```

I think there is something broken.
Using this patch:

```patch
diff --git a/src/lib/Makefile.in b/src/lib/Makefile.in
index 918547ad..e76d8edc 100644
--- a/src/lib/Makefile.in
+++ b/src/lib/Makefile.in
@@ -91,8 +91,9 @@ cppcheck:
        cppcheck --template='{file}:{line}:{severity}:{message}' --quiet --enable=all --force -I ../include *.c protocols/*.c
 
 install: $(NDPI_LIBS)
-       mkdir -p $(DESTDIR)$(PREFIX)$(libdir)
-       cp $(NDPI_LIBS) $(DESTDIR)$(PREFIX)$(libdir)/
+       mkdir -p $(DESTDIR)$(libdir)
+       cp $(NDPI_LIBS) $(DESTDIR)$(libdir)/
+
        cp -P $(NDPI_LIB_SHARED_BASE) $(DESTDIR)$(PREFIX)$(libdir)/
        cp -P $(NDPI_LIB_SHARED_BASE).$(NDPI_VERSION_MAJOR) $(DESTDIR)$(PREFIX)$(libdir)/
        mkdir -p $(DESTDIR)$(PREFIX)$(includedir)
```

and I get this error:

```txt
Making install in src/lib
make[1]: Entering directory '/home/toni/git/nDPI/src/lib'
mkdir -p /home/toni/git/nDPI/tmp_install/lib
cp libndpi.a libndpi.so.4.9.0 /home/toni/git/nDPI/tmp_install/lib/
cp -P libndpi.so /home/toni/git/nDPI/tmp_install/usr/bla/lib/
cp: cannot create symbolic link '/home/toni/git/nDPI/tmp_install/usr/bla/lib/': No such file or directory
make[1]: *** [Makefile:96: install] Error 1
make[1]: Leaving directory '/home/toni/git/nDPI/src/lib'
make: *** [Makefile:557: install-recursive] Error 1
```

while configuring and building it with:

```shell
./autogen.sh --prefix=/usr/bla && make install DESTDIR=$(realpath ./tmp_install)
```

Does it work for you or do I miss something?

--- a/src/lib/Makefile.in
+++ b/src/lib/Makefile.in
@@ -11,9 +11,12 @@ RANLIB = @RANLIB@
 #
 # Installation directories
 #
-PREFIX     = @prefix@
+prefix = @prefix@
+exec_prefix = @exec_prefix@
 libdir     = @libdir@
 includedir = @includedir@/ndpi
+# For backwards compatibility
+PREFIX = $(prefix)
 ifneq ($(OS),Windows_NT)
 OS := $(shell uname)
 endif
@@ -96,10 +99,10 @@ cppcheck:
 	cppcheck --template='{file}:{line}:{severity}:{message}' --quiet --enable=all --force -I ../include *.c protocols/*.c
 
 install: $(NDPI_LIBS)
-	mkdir -p $(DESTDIR)$(PREFIX)$(libdir)
-	cp $(NDPI_LIBS) $(DESTDIR)$(PREFIX)$(libdir)/
-	cp -P $(NDPI_LIB_SHARED_BASE) $(DESTDIR)$(PREFIX)$(libdir)/
-	cp -P $(NDPI_LIB_SHARED_BASE).$(NDPI_VERSION_MAJOR) $(DESTDIR)$(PREFIX)$(libdir)/
-	mkdir -p $(DESTDIR)$(PREFIX)$(includedir)
+	mkdir -p $(DESTDIR)@libdir@
+	cp $(NDPI_LIBS) $(DESTDIR)@libdir@/
+	cp -P $(NDPI_LIB_SHARED_BASE) $(DESTDIR)@libdir@/
+	cp -P $(NDPI_LIB_SHARED_BASE).$(NDPI_VERSION_MAJOR) $(DESTDIR)@libdir@/
+	mkdir -p $(DESTDIR)@includedir@/ndpi
 	#Avoid installing private header
-	find ../include/*.h ! -name ndpi_private.h -exec cp "{}" $(DESTDIR)$(PREFIX)$(includedir)/ \;
+	find ../include/*.h ! -name ndpi_private.h -exec cp "{}" $(DESTDIR)@includedir@/ndpi/ \;

--- a/src/lib/Makefile.in
+++ b/src/lib/Makefile.in
@@ -103,6 +103,6 @@ install: $(NDPI_LIBS)
 	cp $(NDPI_LIBS) $(DESTDIR)@libdir@/
 	cp -P $(NDPI_LIB_SHARED_BASE) $(DESTDIR)@libdir@/
 	cp -P $(NDPI_LIB_SHARED_BASE).$(NDPI_VERSION_MAJOR) $(DESTDIR)@libdir@/
-	mkdir -p $(DESTDIR)@includedir@/ndpi
+	mkdir -p $(DESTDIR)@includedir@
 	#Avoid installing private header
-	find ../include/*.h ! -name ndpi_private.h -exec cp "{}" $(DESTDIR)@includedir@/ndpi/ \;
+	find ../include/*.h ! -name ndpi_private.h -exec cp "{}" $(DESTDIR)@includedir@/ \;

